pkg/tsdb/cloudwatch/models/request_parser.go

```diff
@@ -45,9 +45,10 @@ type metricsDataQuery struct {
 	Alias             string                 `json:"alias,omitempty"`
 }
 
-// ParseQueries parses the json queries and returns a map of cloudWatchQueries by region. The cloudWatchQuery has a 1 to 1 mapping to a query editor row
-func ParseQueries(queries []backend.DataQuery, startTime time.Time, endTime time.Time, dynamicLabelsEnabled bool) (map[string][]*CloudWatchQuery, error) {
-	result := make(map[string][]*CloudWatchQuery)
+// ParseQueries decodes the metric data queries json, validates, sets default values and returns an array of CloudWatchQueries.
+// The CloudWatchQuery has a 1 to 1 mapping to a query editor row
+func ParseQueries(queries []backend.DataQuery, startTime time.Time, endTime time.Time, dynamicLabelsEnabled bool) ([]*CloudWatchQuery, error) {
```

Comment:
Maybe we should call this `ParseCloudWatchQueries`?

Reply:
I was originally going to go with something like `ParseMetricDataQueries` to say what we are parsing. 
Would you prefer it to refer to the output? How about`TransformToCloudWatchQueries`?
`ParseMetricDataQueriesToCloudWatchQueries` ? 

Or go with your suggestion here?

Reply:
Ah no I think your suggestion, `ParseMetricDataQueries` is best!